File: LeetCode/Java/1748. Sum of Unique Elements.java
class Solution {
    public int sumOfUnique(int[] nums) {
        HashMap<Integer, Integer> counter = new HashMap<>();
        int sum = 0;
        
        for (int num: nums)
            counter.put(num, counter.getOrDefault(num, 0) + 1);
        
        for (Map.Entry<Integer, Integer> pair : counter.entrySet())
            if (pair.getValue() == 1)
                sum += pair.getKey();
        
        return sum;
    }
}

File: LeetCode/Python/1748. Sum of Unique Elements.py
class Solution:
    def sumOfUnique(self, nums: List[int]) -> int:
        from collections import Counter
        
        c = Counter(nums)
        _sum = 0
        
        for key in c:
            if c.get(key) == 1:
                _sum += key
        
        return _sum
